package.json

```diff
@@ -1,12 +1,49 @@
 {
-    "name": "mould",
-    "version": "0.1.0",
-    "private": true,
+    "name": "@mouldjs/mould",
+    "version": "0.1.4",
     "scripts": {
         "dev": "node scripts/ensure-test-workdir && cross-env WORKDIR=./test-mould next dev",
-        "build": "next build",
-        "start": "next start"
+        "clean": "rm -rf release/*",
+        "postbuild": "copyfiles -e \"./node_modules/**/*\" -s \"./**/*\" release",
```

Comment:
imho, we still need to `rm -rf release/.mould release/test-mould`, because devs could have this directories while development, but we don't wanna publish them. What do you think?

Reply:
these directories are not included in `package.json:fiels`, it won't be added

Reply:
Yeah, I agree. So we have a white list 'package.files' already, folders like 'test-mould' are outside the list so that they won't be published